pmm-app/src/pmm-remote-instances-panel/module.ts

```diff
@@ -0,0 +1,24 @@
+/// <reference path="../../headers/common.d.ts" />
+
+import {MetricsPanelCtrl} from 'app/plugins/sdk';
+import config from 'app/core/config';
+
+export class PanelCtrl extends MetricsPanelCtrl {
+
+    static template = `
+		<iframe ng-src="{{ trustSrc(url) }}"
+			style="width: 100%; height: 400px; border: 0;" scrolling="no" />
+	`;
+
+    base_url = '/qan/pmm-list';
```

Comment:
base_url = '/graph/d/pmm-list';

Could you check if it's used the correct location of the page?